test/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+cmake_minimum_required(VERSION 3.14 FATAL_ERROR)
+
+project(LittleTypeLibraryTests LANGUAGES CXX)
+
+# ---- Add root project ----
+
+# We want warnings in tests
+set(LittleTypeLibrary_INCLUDE_WITHOUT_SYSTEM ON CACHE INTERNAL "")
+
+include(FetchContent)
+FetchContent_Declare(LittleTypeLibrary SOURCE_DIR "${CMAKE_CURRENT_LIST_DIR}/..")
+FetchContent_MakeAvailable(LittleTypeLibrary)
```

Comment:
Is the MakeAvailable needed here?

In the documentation, it is wrote : `the FetchContent module makes content available immediately`

Btw, thanks for your PR, I am going to accept it this evening :)

Reply:
That part of the documentation just means that `FetchContent` runs in the configuration step, as opposed to `ExternalProject`, which runs in build step.

`FetchContent_Declare(X Y...)` merely declares `X` dependency with `Y` properties and `FetchContent_MakeAvailable` is actually what imports the listed dependencies.